src/main/groovy/com/pedjak/gradle/plugins/dockerizedtest/DockerizedTestPlugin.groovy

```diff
@@ -157,13 +186,23 @@ class DockerizedTestPlugin implements Plugin<Project> {
             {
                 if (address == null)
                 {
-                    def remoteAddresses = NetworkInterface.networkInterfaces.findAll { it.up && !it.loopback }*.inetAddresses*.collect { it }.flatten()
+                    def remoteAddresses = NetworkInterface.networkInterfaces.findAll {
```

Comment:
This change is unrelated to Gradle 4.8, therefore I would like to add it in a separate PR.